modules/porous_flow/doc/content/source/materials/PorousFlowHystereticInfo.md

```diff
@@ -0,0 +1,29 @@
+# PorousFlowHystereticInfo
```

Comment:
This material is only used to generate plots to see what the hysteric curves look like?

Reply:
Yea, basically that's right.  Partly i needed something that could test stuff, but also i believe users might find it useful to explore hysteresis before actually implementing it in models.

Reply:
I added a little note to this effect.